### PR TITLE
Checkbox24 for armhf via remote builds (infra)

### PR DESCRIPTION
**Pull request by Hook25**
## Description

TLDR: This introduces a pipeline to build checkbox24 on armhf. While the other checkbox24 we can build natively, this we have to dispatch to launchpad so that it is built on armhf.

This build runtime and frontend using remote-build because we have no armhf shr. The new remote-build works differently, namely it requires a clean branch to push on launchpad. This is an issue for us because all the prepare.sh scripts copy around files to create the build directory. This introduces a new trio of prepare scripts that do the same thing but copying "additional files" (namely: the common files and the recipe) to the root of the repository. Those changes are then automatically committed by the pipeline so that the branch is "clean".

## Resolved issues

N/A

## Documentation

N/A

## Tests

The version currently in the store was built with this pipeline (although due to an issue it was pushed manually)

### CODE REVIEW

**Review by fernando79513 — APPROVED** [review]
LGTM +1!

**Review by fernando79513 — APPROVED** [review]
LGTM +1!
